danewalton/Microchip-SAM-IoT-Wx-1
# Dedicated Telemetry Interface (DTI)

## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Use a PIC-IoT board to act as the Application MCU in SPI Master mode.  Reference slide #17 in the PPT file specification to see how to connect the PIC-IoT board to the SAM-IoT board using their respective mikroBUS connectors.  Each time the SW0 button is pressed on the PIC-IoT board, a data frame of 1KB payload will be output via SPI
```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
